oak/server/rust/oak_runtime/src/node/mod.rs

```diff
@@ -0,0 +1,64 @@
+use log::info;
+
+use alloc::string::String;
+use alloc::sync::Arc;
+
+use crate::channel::ChannelReader;
+use crate::Runtime;
+
+use oak_abi::OakStatus;
+
+mod wasm;
+
+#[derive(Clone)]
+/// TODO: doc
+pub enum NodeConfiguration {
+    LogNode,
+    WasmNode { module: Arc<wasmi::Module> },
```

Comment:
Why does the module need to be behind an `Arc`? I would have thought that the node owns it fully?